loader/sky-processor/index.js

```diff
@@ -0,0 +1,50 @@
+/*jshint node: true*/
+'use strict';
+
+const logger = require('winston');
+const skyPagesConfigUtil = require('../../config/sky-pages/sky-pages.config');
+const config = skyPagesConfigUtil.getSkyPagesConfig();
```

Comment:
Didn't realize the config was baked into the loader. Changes made, @Blackbaud-BobbyEarl